### DISCUSSION TITLE
Can someone provide me with the CodeCanyon WordPress theme development standards guide?

### DISCUSSION THREAD

**Discussion by MdRaisulIslam02** [community/community#134048] [Programming Help] [answered]
### Body

Can someone provide me with the CodeCanyon WordPress theme development standards guide?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MdRasel5** [comment, marked as answer]
You can follow this: https://help.author.envato.com/hc/en-us/articles/360000472383-WordPress-Theme-Requirements-Start-here
